CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 ### [Versions](https://material-ui.com/versions/)
 
+## 4.12.3
+
+<!-- generated comparing v4.12.2..master -->
+
+_Jul 27, 2021_
+
+### `@material-ui/core@4.12.3`
+
+- <!-- 2 --> [AccordionSummary] Fix false-positive propType warning with `disableGeneration` (#27385) @eps1lon
+- <!-- 6 --> [TablePagination] Re-introduce deprecated `onChangePage` to `ActionsComponent` (#27407) @eps1lon
```

Comment:
The component is the Table?

```suggestion
- <!-- 6 --> [Table] Re-introduce deprecated TablePagination `onChangePage` to `ActionsComponent` (#27407) @eps1lon
```

Reply:
We changed `TablePagination`, not `Table`:
 https://github.com/mui-org/material-ui/pull/27407/files

Reply:
I had tried to document the standard used so far in https://www.notion.so/material-ui/GitHub-issues-PRs-c1d7072e0c2545b0beb43b115f6030f6

> An issue most often has one primary label. It's defined with the "[XXX] Syntax". XXX must match with existing GitHub labels.

It's arbitrary, I think that what matters is to be consistent.

If we want to change the standard, then it's a different discussion.

Reply:
We didn't discuss it to begin with so I don't accept that standard. Your approach is not suitable for a changelog which is what we're discussing here.